sdk/textanalytics/azure-ai-textanalytics/azure/ai/textanalytics/_generated/v3_1_preview_1/__init__.py

```diff
@@ -0,0 +1,16 @@
+# coding=utf-8
```

Comment:
Do we need the gen code for 3.1-preview.1? @iscai-msft

Reply:
nah we don't, thanks @kristapratico for catching. @abhahn can you just generate with versions 3.0, 3.1-preview.2, and 3.1-preview.3, since preview 1 isn't needed? thanks!

Reply:
Ok, I've removed the v3.1-preview.1 folder since it looks like it's not there on master.